packages/gravity-ui-nunjucks/gulp/color-scheme-tables.js

```diff
@@ -18,7 +19,7 @@ function wcagRating(contrastRatio) {
 }
 
 function gravityCssCustomPropName(group, colorName) {
-  return `--grav-co-grp-${group}-${kebabCase(colorName)}`;
+  return `--grav-co-grp-${group}-${kebabCase(startCase(colorName))}`;
```

Comment:
Is there a benefit of chaining `kebabCase()` and `startCase()` here?

Reply:
Nope. I've killed it now

I _think_ I was having trouble with some names getting mangled and that's why I originally did this. However, it seems to work fine without now.